modules/collect/src/main/java/com/opengamma/strata/collect/result/ValueWithFailures.java

```diff
@@ -87,6 +89,27 @@
   }
 
   //-------------------------------------------------------------------------
+
+  /**
+   * Returns a {@link Collector} which performs a reduction of its
+   * {@link ValueWithFailures} input elements under a specified {@link BinaryOperator} using the
+   * provided identity.
```

Comment:
In Strata we avoid using `@link` in the first sentence as it appears in higher-level Javadoc.

```
 * Returns a collector that can be used to create an instance from a stream of amounts.
 * <p>
 * <your more detailed description>
```